src/backend/bs_publish

```diff
@@ -1521,6 +1521,9 @@ sub publish {
 	if ($bin =~ /\.iso(?:\.sha256)?$/) {
 	  $p = "iso/$bin";
 	  $kiwimedium{$p} = $1 if $bin =~ /(.+)\.iso$/;
+	} elsif ($bin =~ /ONIE\.bin$/) {
```

Comment:
this should be \.ONIE\.bin$ since it is a suffix?

Reply:
It gets appeneded without a . in the tools I've seen, eg:

```vyatta-vrouter-17.2-Update_20180219T1526-amd64-Build15.303.iso-ONIE.bin```

not my favourite choice of format :-)